src/s3a_the_gutter.tw2

```diff
@@ -1,174 +1,913 @@
+
+[[“Go for it, Donnie! Blast that shit!”|The Gutter - Jukebox - Song finale]]
+
+::The Gutter - Jukebox - Saying hey
+"What up Donnie?!" I screamed across the bar and raised the horns.
+
+Donnie raised them back and we [[exchanged a righteous nod|The Gutter - Inside hub]]. That kid was alright.
+
+::The Gutter - Bathroom
+Fuck this place, fuck this smoke, fuck these people.
+
+I tromped through the bar's blue clouds and gave the door to the 'Lettes room two sharp raps before barreling in. With a crowd like the one out there, it was pretty unlikely that anyone was inside.
+
+And yup — as I turned the handle and swung the door, the bathroom stood empty. I closed the door tight behind me and kicked the stopper in under the bottom edge.
+
+Next best thing to a lock if you were cheap as fuck. And anyway, bartender still needed to be able to make sure ninjas weren't getting wet and wild on the fucking shitter.
+
+(click: "wet and wild on the fucking shitter")[As soon as I turned away from the door, I (live: 1s)[(if: time < 20s)[$dissolve[[[started puking into the toilet|The Gutter - Bathroom - Toilet puke]]]]
+\(else-if: time > 20s)[$dissolve[[[started puking on myself|The Gutter - Bathroom - Sink puke]](stop:)]]].]
+
+(set: $mirrorSeen to false)(set: $toiletflashbackSeen to false)(set: $bathroomVisited to true)
+
+::The Gutter - Bathroom - Toilet puke
+I barely propped the toilet lid open fast enough to hit the mark. I barfed a strong, hateful stream into the bowl.
+
+When I was finished I was gasping, like I’d been punched in the gut. This night was all too much. Anna gone, maybe dead already. I’ve been drugged, chased, and now I’ve gotta ask my questions with puke on my breath.
+
+I wiped my mouth with the back of my hand. When this was over, I was gonna need a high the length of I-55 to deal.
+
+(display: "The Gutter - Bathroom - Menu")
+
+::The Gutter - Bathroom - Sink puke
+I spewed all over the floor, splashing on my feet, before I finally found the sink. I nearly filled it up with whatever needed out of me.
+
+When I was finished I was gasping, like I’d been punched in the gut. This night was all too much. Anna gone, maybe dead already. I’ve been drugged, chased, and now I’ve gotta ask my questions with puke on my shoes.
+
+I wiped my mouth with the back of my hand. When this was over, I was gonna need a high the length of I-55 to deal.
+
+(display: "The Gutter - Bathroom - Menu")
+
+(set: $sinkPuke to true)
+
+::The Gutter - Bathroom - Menu
+>>[[I headed back out into the din and dipshits|The Gutter - Inside hub]]
+(if: $mirrorSeen is false)[>>[[I was drawn to my reflection in the mirror|The Gutter - Bathroom - Mirror]].]
+(if: $toiletflashbackSeen is false)[>>[[I sat on the toilet to catch my breath|The Gutter - Bathroom - Toilet flashback]].]
+
+::The Gutter - Bathroom - Mirror
+(set: $mirrorSeen to true)
```

Comment:
Same thing here — `\` at the end of this line to make sure the following line appears at the top of the screen.